app/models/user.rb

```diff
@@ -198,6 +198,45 @@ def after_confirmation
     OnboardingMailer.with_user_locales(users: [self], school: school) { |mailer| mailer.welcome_email.deliver_now } if self.school.present?
   end
 
+  def self.admin_user_export_csv
+    CSV.generate do |csv|
+      csv << [
+        'School Group',
+        'School',
+        'School type',
+        'Funder',
+        'Region',
+        'Name',
+        'Email',
+        'Role',
+        'Staff Role',
+        'Locked'
+      ]
+      where.not(role: :pupil).where.not(role: :admin).order(:email).each do |user|
```

Comment:
Could this be shortened to:
`where.not(role: [:pupil, :admin]).order(:email).each do |user|`